tests/manual/monkey/src/com/oracle/tools/fx/monkey/options/FontOption.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) 2023, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
The code also lives in a separate repository where it's copyrighted correctly (since I constantly add things to it), so no change.

https://github.com/andy-goryachev-oracle/MonkeyTest